examples/demo_astradb.ipynb

```diff
@@ -0,0 +1,274 @@
+{
```

Comment:
<p>would be good to have a section looking at one of the source nodes: <code>print(response.source_nodes[0].get_content())</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_parse/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>